src/timeline/TimelineViewManager.h

```diff
@@ -31,7 +32,7 @@ class DeviceVerificationList : public QObject
         Q_INVOKABLE bool exist(QString tran_id);
 
 private:
-        QVector<QString> dv_list;
+        QLinkedList<QString> dv_list;
```

Comment:
I bet that this is actually slower. A vector is usually an order of magnitude faster when deleting and the size is less than a few thousand elements, since everything is in cache. Also, can you write the name out please, dv is not obvious without context.

Reply:
Lol didn't know that then I'll change it back

Reply:
Here are some benchmarks for that: https://baptiste-wicht.com/posts/2012/12/cpp-benchmark-vector-list-deque.html

Basically the algorithmic performance of a list is pretty good, but that's not everything. If you first need to find an element, the pointer chasing of a linked list absolutely destroys the algorithmic benefits. So in general, a std::vector is the best data structure unless your benchmarks show something else. Always prove, that a vector is not the fastest option, never rely on intuition for performance optimizations.